packages/api/src/feature/activity-certificate/controller/activity-certificate.controller.ts

```diff
@@ -0,0 +1,24 @@
+import { Controller, Post, Body, UsePipes } from "@nestjs/common";
+import { ZodPipe } from "@sparcs-clubs/api/common/pipes/zod-pipe";
+import apiAcf001, {
+  ApiAcf001RequestBody,
+  ApiAcf001ResponseCreated,
+} from "@sparcs-clubs/interface/api/activity-certificate/endpoint/apiAcf001";
+import { ActivityCertificateService } from "../service/activity-certificate.service";
+
+@Controller()
+export class ActivityCertificateController {
+  constructor(
+    private readonly activityCertificateService: ActivityCertificateService,
+  ) {}
+
+  // TODO: Implement zod validation
+  @Post("/student/activity-certificates/activity-certificate")
+  @UsePipes(new ZodPipe(apiAcf001))
+  async postActivityCertificate(
+    @Body() body: ApiAcf001RequestBody,
+  ): Promise<ApiAcf001ResponseCreated> {
+    this.activityCertificateService.postActivityCertificate(body);
```

Comment:
```suggestion
    await this.activityCertificateService.postActivityCertificate(body);
```

❗️await 하지 않으면 Exception throw시 exception filter가 catch하지 못하는 경우가 있더라고요!